constellation-app/constellation
# Fix some help buttons not directing to correct page

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change



Fixed bug that made some help buttons direct to the default help page, instead of the correct associated page. This was because help page paths were wrong or were changed at some point.

### Alternate Designs


N/A

### Why Should This Be In Core?


N/A

### Benefits


Help buttons now function correctly.

### Possible Drawbacks


None

### Verification Process



1. Open Consty
2. Open or create a graph
3. Arrange -> BubbleTree3D
4. Click "Help"
5. Observe the "Bubble Tree 3D" was opened

### Applicable Issues


https://github.com/constellation-app/constellation/issues/2217
